wm_2dnav/config/base_local_planner_params.yaml

```diff
@@ -9,20 +9,20 @@ TrajectoryPlannerROS:
   acc_lim_x: 1.0
   acc_lim_y: 1.0
 
-  escape_vel: -0.2
+  escape_vel: 0.2
 
-  holonomic_robot: false
-  # y_vels: [-0.4, -0.1, 0.1, 0.4]
+  holonomic_robot: true
```

Comment:
Are those change related to the new repo structure or have been introduced by mistake?